product_docs/docs/edbcloud/beta/administering_cluster/organization_and_access/02_users.mdx

```diff
@@ -1,24 +1,91 @@
 ---
-title: "Users"
+title: "Managing Users and Access"
```

Comment:
I'm not sure if this should be combined or put back to 2 pages

Reply:
This makes sense to me

Reply:
I think we should keep them as separate topics: Managing Users and Managing Access or Managing Roles and Permissions